prow/slack/reporter/reporter.go

```diff
@@ -77,7 +78,7 @@ func (sr *slackReporter) GetName() string {
 }
 
 func (sr *slackReporter) ShouldReport(pj *v1.ProwJob) bool {
-	config := sr.config()
+	config := sr.config(pj.Spec.Refs)
```

Comment:
Same here, we may get a nilpointer back

Reply:
Adding the same error handling as other reference.